index.html

```diff
@@ -1262,6 +1278,19 @@ <h2>
 controller of the corresponding private key, MUST be a valid DID.
       </p>
 
+      <p class="note" title="This is not the DID Controller">
+The <code>controller</code> property denotes a <a>DID controller</a>
+only when it appears at the <em>top level</em> of the <a>DID document</a>.
+The <code> controller</code> property when it appears as part of a <a>public
+key description</a> denotes the controller of the <em>corresponding
+private key</em> only, which may or may not be described in another
+<a>DID document</a>. A default <code>controller</code> field cannot
+be inferred by using a key's position in a tree, and so is explicitly
+included for each key. Linked Data Proof libraries typically expect the
+<code>controller</code> field to always exist and may throw an exception
+if it is missing.
```

Comment:
The default controller for a key would be itself (the "subject" of the key) just like it is with a DID Document and the DID subject. Since a key can't control itself and the controller of the key should (typically) be the DID subject, the `controller` field must be specified.

I don't think we need to say that Linked Data Proof libraries expect it to exist ... in fact, if they do, that should be fixed so that they assume the controller is the key itself if it is not present. Regardless, keys in a DID document should always have a `controller` because they are controlled by the DID subject (or, transitively, the DID controller if it is not the same as the DID subject).

Reply:
Yeah, some of that was old wording that I thought was a bit odd. I agree with taking the ldproof part out. `controller` is already a MUST for keys, so I don't think we need to say anything about a default controller, unless that MUST ever comes down to a SHOULD, because it's always explicit otherwise.

Reply:
I think it may also be useful to show that `controller` is being reused with the same semantics as when it appears in the DID Document. The `controller` of a thing/object is specified by the `controller` property -- and since a key can't control itself (it has no keys of its own for example), we want to express that keys are controlled by a DID subject (again, or transitively that DID subject's DID controller).

Reply:
I misread what you said initially but yeah I agree

Reply:
That reminds me, can a key have more than one controller? I think we need to explicitly state that it can be an array too right?

Reply:
Is it true that there could be a case where a key controller is not the DID controller for that DID, but some other entity?

Reply:
> That reminds me, can a key have more than one controller? 

No, it can't.

> I think we need to explicitly state that it can be an array too right?

No, I don't think controller can ever be an array, IIRC.